nelua/cdefs.lua

```diff
@@ -189,9 +192,90 @@ cdefs.target_info_code = [[
 #if defined(__ANDROID__)
   is_android = true;
 #endif
+#if defined(APPLE_IOS) || defined(IOS) || defined(__IPHONEOS__) || defined(TARGET_OS_SIMULATOR) || defined(__IOS__) || defined(__APPLE_IOS__) || defined(IOS_OBJ_C) || defined(TARGET_OS_IOS)
+  is_ios = true;
+#endif
 #if defined(__MACH__)
   is_mach = true;
 #endif
+#if defined(__HPUX__)
+  is_hpux = true;
+#endif
+#if defined(__SGI__)
+  is_irix = true;
+#endif
+#if defined(__TIZEN_H__) || defined(TIZEN_DEPRECATION) || defined(__TIZEN__)
+  is_tizen = true;
+#endif
+#if defined(__GAMEBOY__) || defined(__gba__) || defined(__GBA__)
+  is_gameboy = true;
+#endif
+#if defined(_NDS) || defined(NDS) || defined(__NDS__) || defined (__NDS)
```

Comment:
Remove the `NDS`, others defines should do the job, defines without `_ `prefix is usually a not good ideia to use on detection, because it may be defined by user headers, unless you are sure this is the only way to detect.

Reply:
Removed!